FiveCalls/FiveCalls/IssuesViewController.swift

```diff
@@ -105,8 +105,8 @@ class IssuesViewController : UITableViewController {
         isLoading = true
         tableView.reloadEmptyDataSet()
         issuesDelegate?.didStartLoadingIssues()
-        let query: IssuesManager.Query = shouldFetchAllIssues ? .all : .top(UserLocation.current)
```

Comment:
and this line could probably have been improved as a separate method that returns the appropriate `Query` case.